ui/src/me/components/account/Settings.tsx

```diff
@@ -54,12 +56,20 @@ export class Settings extends PureComponent<StateProps, State> {
                 </Form>
               </Panel.Body>
             </Panel>
+            <ColorPicker
+              selectedHex={color}
+              onSelect={this.handleSelectColor}
+            />
           </Grid.Column>
         </Grid.Row>
       </Grid>
     )
   }
 
+  private handleSelectColor = (color: string): void => {
+    this.setState({color})
+  }
+
   private handleChangeInput = (_: ChangeEvent<HTMLInputElement>): void => {
     //  console.log('changing: ', e)
```

Comment:
could this console.log commetn be removed?

Reply:
I think this is actually in there on `master`. It makes me wonder if the settings page is even doing anything